internal/services/v1/relationships.go

```diff
@@ -126,30 +130,49 @@ func (ps *permissionServer) ReadRelationships(req *v1.ReadRelationshipsRequest,
 		return rewriteError(ctx, err)
 	}
 
-	usagemetrics.SetInContext(ctx, &dispatchv1.ResponseMeta{
-		DispatchCount: 1,
-	})
+	batchCount := datastoreQueryBatchSize
+	var cursor options.Cursor
+	var dispatchCount uint32
+	for batchCount == datastoreQueryBatchSize {
```

Comment:
This patterns will become useful in other spots as we move different parts of the codebase to pagination. I could see us creating a `RelationshipIterator` that abstracts the need to fetch the cursor and infinitely scrolls over the pages. WDYT?

Reply:
+1 to that idea

Reply:
Done.